input_grid_map_topic: "/elevation_mapping/elevation_map_recordable" # topic of subscription for input grid maps
occ_grid_map_topic: "occ_grid_map" # topic of publication of occluded grid maps
rec_grid_map_topic: "rec_grid_map" # topic of publication of reconstructed grid maps
comp_grid_map_topic: "comp_grid_map" # topic of publication of composed grid maps
all_grid_map_topic: "all_grid_map" # topic of publication for GridMap messages with all layers

inpaint_method: 2 # Navier-Stokes, Telea or NN
input_layer: "elevation" # in which layer of the GridMap object is the occluded DEM stored

resizing: {
  resize: true,
  target_resolution: 0.046875
}

# OpenCV parameters
inpaint_radius: 0.140625

# neural network parameters
neural_network_path: "models/eth_stairs_traced.pt" # relative to package root
use_gpu: false
NaN_replacement: 0.0

subgrids: {
  divide_into_subgrids: true,
  subgrid_rows: 64,
  subgrid_cols: 64,
  subgrid_max_occ_ratio_thresh: 0.85
}

# visualization
visualize_with_open_cv: false